docs/src/content/en/docs/agents/sanitizing-input.mdx

```diff
@@ -176,59 +176,55 @@ Strategies available:
 - `block`: Reject content not in target language
 - `warn`: Log warning but allow content through
 
-## Applying Multiple Processors
+## Applying multiple processors
 
 You can chain multiple processors. They execute sequentially in the order they appear in the `inputProcessors` array. The output of one processor becomes the input for the next.
 
 **Order matters!** Generally, it's best practice to place text normalization first, security checks next, and content modification last.
 
-```typescript copy showLineNumbers {9-18}
-import { Agent } from "@mastra/core/agent";
+```typescript {10-15} filename="src/mastra/agents/test-agent.ts" showLineNumbers copy
 import {
   UnicodeNormalizer,
   ModerationProcessor,
   PromptInjectionDetector,
   PIIDetector
 } from "@mastra/core/processors";
 
-const secureAgent = new Agent({
+export const testAgent = new Agent({
+  // ...
   inputProcessors: [
```

Comment:
this code example illustrates how order matters, perhaps you can consider moving the note about that here